### PR TITLE
MAD4Jekyll theme

### PR DESCRIPTION
**Pull request by madforjekyll**

### CODE REVIEW

File: _posts/2015-11-25-MAD4Jekyll.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+layout: post
+title: MAD4Jekyll
+date: 2015-11-25 00:00:00
+homepage: https://github.com/madforjekyll/madforjekyll.github.io
+download: https://github.com/madforjekyll/madforjekyll.github.io/archive/master.zip
+demo: madforjekyll.github.io 
+author: MADness
+thumbnail: MAD4jekyll.png
+license: Apache
+license_link: https://github.com/madforjekyll/madforjekyll.github.io/blob/master/LICENSE
```
Comment (gynter) [inline _posts/2015-11-25-MAD4Jekyll.md]: License link is not working.

File: _posts/2015-11-25-MAD4Jekyll.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: MAD4Jekyll
+date: 2015-11-25 00:00:00
+homepage: https://github.com/madforjekyll/madforjekyll.github.io
+download: https://github.com/madforjekyll/madforjekyll.github.io/archive/master.zip
+demo: madforjekyll.github.io 
+author: MADness
+thumbnail: MAD4jekyll.png
+license: Apache
```
Comment (gynter) [inline _posts/2015-11-25-MAD4Jekyll.md]: Hello! Thank You for the contribution. Please change the license to "Apache License Version 2.0". Also please use `.markdown` as extension not `.md`.

File: thumbnails/mad4jekyll [hunk 3]
```diff
@@ -0,0 +1 @@
+![mad4jekyll](https://cloud.githubusercontent.com/assets/15498648/11429252/f2ddb7b0-947d-11e5-9259-3efd315f2f6f.png)
```
Comment (gynter) [inline thumbnails/mad4jekyll]: This is not a correct thumbnail. Thumbnail should be in image format (i.e jpg or png) with dimensions 250 x 200.

File: _posts/2015-11-25-MAD4Jekyll.md (hunk 1 above)
Comment (madforjekyll) [inline _posts/2015-11-25-MAD4Jekyll.md]: Corrected!

File: _posts/2015-11-25-MAD4Jekyll.md (hunk 2 above)
Comment (madforjekyll) [inline _posts/2015-11-25-MAD4Jekyll.md]: Corrected!

File: thumbnails/mad4jekyll (hunk 3 above)
Comment (madforjekyll) [inline thumbnails/mad4jekyll]: Corrected!

**Pull request merged**